event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/types/Payload.java

```diff
@@ -249,6 +261,46 @@ public void setLabels(Labels labels) {
         this.labels = labels;
     }
 
+    @JsonProperty("playbook_name")
```

Comment:
I think you did it the other way around. It should be:
* `playbook_name` is the field name in [the database](https://github.com/RedHatInsights/playbook-dispatcher/blob/master/migrations/011_alter_runs_sat-fields.up.sql) (incoming stream)
* `name` is what the field is called [in our APIs](https://github.com/RedHatInsights/playbook-dispatcher/blob/master/schema/private.openapi.yaml#L132) (outgoing stream)

Reply:
Oh whoops! Now I think I understand what's going on here.

Thanks again, @jharting